### ISSUE TITLE
Problem getting preferences back from Application Support/com.tinyapp.TablePlus/Cache/Favorite

### ISSUE THREAD

**Issue by thedoglett** [TablePlus/TablePlus#3298]
My Mac completely died on me and I took the opportunity to do a fresh install of everything, including TablePlus.

I have multiple Time Machine backups and having looked through the issues here I went to the Application Support folder to grab my favourites which should be in:

```
Application Support/com.tinyapp.TablePlus/Cache/Favorite
```

When I go to the Application Support folder from any of my backups I can't see the above 'com.tinyapp.TablePlus'. I've even looked in the Users/me/Library folder and its not in there either.

Has this changed, since the last question I found about it (back in 2019) or is it maybe an issue trying to find it in a Time Machine backup via the Finder?

I don't want to backup from the Time Machine backups as there was something causing my Mac to freeze all the time, plus it would take ages to revert a 1TB drive.

Any help would be appreciated!

**Comment by huyphams** [comment]
Hi @thedoglett we did not change anything, the path remain as 

/Users/<your_user_name>/Library/Application Support/com.tinyapp.TablePlus/Cache/Favorite

**Comment by thedoglett** [comment]
Oh dear, that's worrying. I wonder why Time Machine didn't copy it over.

Problem is no doubt on my side, will have to see what I can do.

Thanks for the heads-up.

**Comment by huyphams** [comment]
Thanks @thedoglett let me know if you have any question via email nick@tableplus.com 
closing issue.

**Comment by arzurchris** [comment]
Hi @thedoglett I'm facing the same issue. Did you find a way to restore your DB from Time Machine backups?

**Comment by thedoglett** [comment]
There was something odd happening when I was trying so what I did was this:

1. Navigated to /Users/<your_user_name> in the Time Machine Backup
2. Held down Shift+Command+. to show invisible files & folders
3. Went into the Library folder and inside Application Support the tinyapp folder was there
4. I was able to grab the favourites and put them onto my machine

Not sure why it wasn't working originally, it was definitely not there, but after doing that I could see it.

Hope that helps.
